SQL/updates/59-60.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE
```

Comment:
Can you include the update numbers (from 59 to 60), a brief description of the script, and your handle here at the top?

Reply:
Done